cob_generic_can/CMakeLists.txt

```diff
@@ -3,14 +3,17 @@ project(cob_generic_can)
 
 find_package(catkin REQUIRED COMPONENTS cob_utilities libntcan libpcan socketcan_interface)
 
+find_package(Boost REQUIRED COMPONENTS chrono)
```

Comment:
`socketcan_interface` was patched in https://github.com/ros-industrial/ros_canopen/commit/bc5e98263c7b6789b33d309771b452b05bed7223